src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -2324,18 +2324,21 @@ internal void TogglePreviewControlAllowance()
         /// </summary>
         private void StashNodeViewCustomizationMenuItems()
         {
-            foreach (var obj in grid.ContextMenu.Items)
```

Comment:
Is `grid.ContextMenu` assigned to `MainContextMenu` somewhere?

Reply:
Okay, I don't understand the logic in that case, but if you're confident about these changes, please feel free to merge.